src/index.html

```diff
@@ -11,7 +11,136 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="breakable">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="Surname"
+          >
+        </label>
+        <label class="breakable">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="Name"
+          >
+        </label>
+        <label class="breakable">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="breakable">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+          >
+        </label>
+        <label class="breakable">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="acceptTerms"
```

Comment:
You can make it in one line if there are < 3 attributes and the line is not too big